index.html

```diff
@@ -1,26 +1,29 @@
 <html>
 
 <body>
-    <div id="title-box-wrapper">
-        <div id="title-box">J-DAWG'S LEGIT 8-BIT MUSIC MAKER 5000</div>
-    </div>
-    </br>
-    <div id="buttons">
-        <button id="start-button">START</button>
-        <button id="stop-button">STOP</button>
-        <button id="play-button">PLAY</button>
-        <button id="new-song-button">New Song</button>
-    </div>
-    </br>
     <div id="notes-and-grid-wrapper">
         <div id="note-name-boxes">
-            <!-- <div id="output"></div> -->
             <br />
         </div>
         <div id="grey-box">
             <!-- <div id="grey-box-grid"></div> -->
         </div>
     </div>
+    <div class="menu">
+        <div id="title-box-wrapper">
+            <div id="title-box">J-DAWG'S BeepBox 2.0</div>
+        </div>
+        <div id="buttons">
+            <button id="start-button">START</button>
+            <button id="stop-button">STOP</button>
+            <button id="play-button">PLAY</button>
+            <button id="new-song-button">New Song</button>
+        </div>
+        <div class="vol-slide-container">
+            <input type="range" min="1" max="100" value="50" class="slider" id="MyRange">
```

Comment:
I'm sensing you copied this input from somewhere -- let's give it a more meaningful id than `MyRange`

Reply:
Yes, I copied it from W3schools lol, I'll change it